fastify.js

```diff
@@ -2,16 +2,19 @@
 
 const wayfarer = require('wayfarer')
 const stripUrl = require('pathname-match')
-const pluginLoader = require('boot-in-the-arse')
+const pluginLoader = require('avvio')
 const http = require('http')
 const https = require('https')
 const pinoHttp = require('pino-http')
 const Middie = require('middie')
-const Reply = require('./lib/reply')
+const mapSeries = require('steed')().mapSeries
```

Comment:
we should probably want to do this inside `build()`, to limit it to a fastify instance.